db/migrate/20150113143508_add_popular_article.rb

```diff
@@ -0,0 +1,10 @@
+class AddPopularArticle < ActiveRecord::Migration
+  def up
+    create_table :popular_articles do |t|
```

Comment:
It just stores validations. We use the id for the many to many table. Not ideal but this functionality is supposed to be temporary anyway.